### ISSUE TITLE
java.lang.UnsatisfiedLinkError: dlopen failed: library "libc++_shared.so" not found

### ISSUE THREAD

**Issue by jfacoustic** [paamand/GDAL#7]
I'm trying to import GDAL as a module to my project. Here's my code:

```kotlin
val testDS: Dataset = gdal.Open(myApp.getSelectedFile())
Log.e("GDAL Output:", gdal.GDALInfo(testDS, null))
```

It's giving the following output errors:

```
Native library load failed

java.lang.UnsatisfiedLinkError: dlopen failed: library "libc++_shared.so" not found

No implementation found for long org.gdal.gdal.gdalJNI.Open__SWIG_1(java.lang.String) (tried Java_org_gdal_gdal_gdalJNI_Open_1_1SWIG_11 and Java_org_gdal_gdal_gdalJNI_Open_1_1SWIG_11__Ljava_lang_String_2)
```

Is there something I'm missing?

**Comment by jfacoustic** [comment]
I'm also trying to load a Geospatial PDF.  Maybe that's the problem.

**Comment by jfacoustic** [comment]
Current compilation doesn't support pdfs.

**Comment by paamand** [comment]
Never tried PDF's with the library, but your error seemed to be because the library was not included in your app package. Did you try any other library functions with success, or how do we know this is a PDF issue?

**Comment by jfacoustic** [comment]
@paamand 
I'm actually not sure why it couldn't find this library.  I tried it again later and it instead that `gdal.Open(...) must not be null`, nothing about missing that library.  Maybe it was some sort of syncing issue.

I got gdal.Open and gdal.GDALInfo working on a TIFF file, but gdal.Open returns null for PDFs.  

It looks like there are some specific ./configure flags to enable PDFs which require either Poppler, PoDoFo, or PDFium.  I'm not sure how I would include that stuff in the compilation.

In submodules/install.sh I added `--with-pdfium --with-pdfium-lib --with-pdfium-extra-lib-for-test --enable-pdf-plugin` and recompiled, but nothing changed.  Doing so natively on my Mac actually gives an error that it can't find PDFium, which makes sense.
